### PR TITLE
e2e-ci: move some functions to ele-testhelpers

### PR DESCRIPTION
**Pull request by ldevulder**
**NOTE:** https://github.com/rancher-sandbox/ele-testhelpers/pull/1 should be merged before this PR!

### CODE REVIEW

File: tests/e2e/e2e_suite_test.go [hunk 1]
```diff
@@ -94,7 +56,7 @@ func deployOperator(k *kubectl.Kubectl) {
 		err = k.WaitForPod("cattle-rancheros-operator-system", "app=rancheros-operator", "rancheros-operator")
 		Expect(err).ToNot(HaveOccurred())
 
-		waitNamespace("cattle-rancheros-operator-system", "app=rancheros-operator")
+		k.WaitForNamespaceWithPod("cattle-rancheros-operator-system", "app=rancheros-operator")
```
Comment (ldevulder) [inline tests/e2e/e2e_suite_test.go]: I've already added a `err` check, will be pushed with the commit that will upgrade `ele-testhelpers` version.

File: tests/e2e/e2e_suite_test.go [hunk 2]
```diff
@@ -134,7 +96,7 @@ var _ = BeforeSuite(func() {
 		// (no further bootstrap is required)
 		By("Upgrading the operator only", func() {
 			kubectl.DeleteNamespace("cattle-rancheros-operator-system")
```
Comment (ldevulder) [inline tests/e2e/e2e_suite_test.go]: @mudler Is there any reasons why there is no error check in some lines in that code?

Comment (mudler) [reply tests/e2e/e2e_suite_test.go]: No specific reason I think, probably a leftover

Comment (ldevulder) [reply tests/e2e/e2e_suite_test.go]: Ok, I will add some with the next commit.

Comment (ldevulder) [reply tests/e2e/e2e_suite_test.go]: Done!

**Review by mudler — APPROVED** [review]
Looking good! thanks!

**Pull request merged**